tests/ui/unchecked_time_subtraction_unfixable.stderr

```diff
@@ -0,0 +1,29 @@
+error: unchecked subtraction between 'Duration' values
+  --> tests/ui/unchecked_time_subtraction_unfixable.rs:12:13
+   |
+LL |     let _ = dur1 - dur2 - dur3;
+   |             ^^^^^^^^^^^^^^^^^^
+   |
+   = note: `-D clippy::unchecked-time-subtraction` implied by `-D warnings`
+   = help: to override `-D warnings` add `#[allow(clippy::unchecked_time_subtraction)]`
+
+error: unchecked subtraction between 'Duration' values
+  --> tests/ui/unchecked_time_subtraction_unfixable.rs:12:13
+   |
+LL |     let _ = dur1 - dur2 - dur3;
+   |             ^^^^^^^^^^^ help: try: `dur1.checked_sub(dur2).unwrap()`
+
```

Comment:
made this an "unfixable", not sure if this is the right approach. It's nice to _not_ have a suggestion for the `dur - dur - dur`, but have some sort of suggestion come after for the nested? is there a precedent for handling this sort of thing?

Reply:
It's fine to not give a suggestion, but ideally the whole thing would get a single suggestion.